server/src/main/java/org/apache/lucene/search/uhighlight/CustomUnifiedHighlighter.java

```diff
@@ -156,7 +172,7 @@ public Snippet[] highlightField(LeafReader reader, int docId, CheckedSupplier<St
                     + "This maximum can be set by changing the ["
                     + IndexSettings.MAX_ANALYZED_OFFSET_SETTING.getKey()
                     + "] index level setting. "
-                    + "For large texts, indexing with offsets or term vectors is recommended!"
+                    + "For large texts, indexing with offsets or term vectors is recommended! "
```

Comment:
No sure if additional space was intended or not ...

Reply:
No it was not